src/store/uiInteractions.js

```diff
@@ -1,7 +1,8 @@
-import { localStorageWrapper } from '@/main'
+import { LocalStorageWrapper } from '@/api'
 
 export default {
   state: {
+    localStorage: new LocalStorageWrapper(),
```

Comment:
As the `localStorageWrapper` never changes it doesn't need to be inside the state